scripts/scil_correct_gradients.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Validate and correct b-vectors using a fiber coherence index as presented
+in Schilling et al, 2019. The script takes as input the principal direction
```

Comment:
lets call the script scil_validate_and_correct_bvecs.py no?

Reply:
yes sounds good to me!